src/components/ImageGrid/ImageGrid.tsx

```diff
@@ -66,15 +67,28 @@ export const ImageGrid = ({ openDrawer }: ImageGridProps) => {
     return categories[index];
   };
 
+  const scaleFactor = useSelector(tileSizeSelector);
+
+  const getSize = (scaleFactor: number) => {
+    const width = (230 * scaleFactor).toString() + "px";
```

Comment:
Where do the `230` and `180` constants come from?

Reply:
How do you maintain the aspect ratio of the image?

Reply:
Constants came from the current defaults generated by GridView. In the current version (and original?) it looks like the images are scaled and cropped to fit that frame. We can happily adjust this or the underlying scaling/cropping applied before making these viewer windows.